workflows/parameters/GFDL-CM4-tasmin.yaml

```diff
@@ -4,7 +4,7 @@ jobs: |
       "target": "historical",
       "variable_id": "tasmin",
       "historical": { "activity_id": "CMIP", "experiment_id": "historical", "table_id": "day", "variable_id": "tasmin", "source_id": "GFDL-CM4", "institution_id": "NOAA-GFDL", "member_id": "r1i1p1f1", "grid_label": "gr1", "version": "20180701" },
-      "ssp": { "activity_id": "ScenarioMIP", "experiment_id": "ssp370", "table_id": "day", "variable_id": "tasmin", "source_id": "GFDL-CM4", "institution_id": "NOAA-GFDL", "member_id": "r1i1p1f1", "grid_label": "gr1", "version": "20180701" }
+      "ssp": { "activity_id": "ScenarioMIP", "experiment_id": "ssp245", "table_id": "day", "variable_id": "tasmin", "source_id": "GFDL-CM4", "institution_id": "NOAA-GFDL", "member_id": "r1i1p1f1", "grid_label": "gr1", "version": "20180701" }
```

Comment:
same q as for tasmax GFDL-CM4

Reply:
@dgergel I didn't see a record for GFDL-CM4 ssp370 (even in the noQC catalog inventory).

Reply:
Ohh got it. This sounds good then. Thanks!